### PR TITLE
fix(custom widget) warn about widget ID restriction

### PR DESCRIPTION
**Pull request by julienmege**
closes[BS-18256](https://bonitasoft.atlassian.net/browse/BS-18256)

### CODE REVIEW

File: md/custom-widgets.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ For more information, see the AngularJS guide on [templates](https://docs.angula
 
 For now, the definition of custom containers is not yet available in the product.
 
+::: warning
+Note that UI Designer always prefix the ID of the custom widget by "custom",
```
Comment (abirembaut) [inline md/custom-widgets.md]: widget**s**

File: md/custom-widgets.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ For more information, see the AngularJS guide on [templates](https://docs.angula
 
 For now, the definition of custom containers is not yet available in the product.
 
+::: warning
+Note that UI Designer always prefix the ID of the custom widget by "custom", 
+whereas the [standard widgets](widgets.md) are internally using the prefix "pb".
+If you build custom widgets outside of the UI Designer, you must not use "pb" as widget ID prefix.
```
Comment (abirembaut) [inline md/custom-widgets.md]: rather
", your widget ID must not start with "pb"."

**Review by abirembaut — COMMENTED** [review]
LGTM. Just 2 comments

File: md/custom-widgets.md (hunk 2 above)
Comment (julienmege) [reply md/custom-widgets.md]: done

**Comment by julienmege** [comment]
done

File: md/custom-widgets.md [hunk 3]
```diff
@@ -21,6 +21,12 @@ For more information, see the AngularJS guide on [templates](https://docs.angula
 
 For now, the definition of custom containers is not yet available in the product.
 
+::: warning
+Note that UI Designer always prefix the ID of the custom widgets by "custom", 
+whereas the [standard widgets](widgets.md) are internally using the prefix "pb".
+If you build custom widgets outside of the UI Designer, your widget ID must not start with "pb".
```
Comment (abirembaut) [inline md/custom-widgets.md]: Maybe (since the second part of the sentence is singular)
"If you build **a widget** outside..."

**Pull request merged**